src/mapping/bd_rhapsody_wta_1_10_1/script.py

```diff
@@ -252,12 +254,27 @@ def strip_margin(text):
 
     print("> " + ' '.join(cmd))
 
-    p = subprocess.Popen(
+    p = subprocess.check_call(
       cmd,
       cwd=os.path.dirname(config_file),
       env=env
     )
-    p.wait()
 
-    if p.returncode != 0:
-      raise Exception(f"cwl-runner finished with exit code {p.returncode}") 
+  # look for counts file
+  if not par["run_name"]:
+    par["run_name"] = "sample"
+  counts_filename = par["run_name"] + "_RSEC_MolsPerCell.csv"
+  
+  if par["sample_tags_version"]:
+    counts_filename = "Combined_" + counts_filename
+  counts_file = os.path.join(par["output"], counts_filename)
+  
+  if not os.path.exists(counts_file):
+    raise ValueError(f"Could not find output counts file '{counts_filename}'")
```

Comment:
Should perhaps become [FileNotFoundError](https://docs.python.org/3/library/exceptions.html#FileNotFoundError)